.travis.yml

```diff
@@ -34,12 +34,8 @@ before_install:
       fi;
     # Only install these deps if necessary (slow on Py3k)
     - travis_retry pip install -q nose-timer flake8;
-    - if [ "${PYTHON}" == "3.4" ]; then
-        travis_retry pip install -q pyglet;
-      else
-        travis_retry pip install -q http://bitbucket.org/pyglet/pyglet/get/tip.zip;
-      fi;
-    # Full dependencies (pandas, mne-python, h5py, joblib)
+    - travis_retry pip install -q pyglet mne;  # mne is a dependency now that resample is taken from it
+    # Full dependencies (pandas, mne-python, h5py, joblib, mne-python)
```

Comment:
since you're mucking around in this file, note the duplicated "mne-python" in this comment